services/QuillLMS/client/app/bundles/Staff/components/comprehension/activities.tsx

```diff
@@ -2,18 +2,22 @@ import * as React from "react";
 import { Link } from 'react-router-dom';
 import { useQuery } from 'react-query';
 
-import Navigation from './navigation'
+import Navigation from './navigation';
 
 import { ActivityInterface } from '../../interfaces/comprehensionInterfaces';
 import { fetchActivities } from '../../utils/comprehension/activityAPIs';
 import { DataTable, Error, Spinner } from '../../../Shared/index';
+import { handleHasAppSetting } from "../../../Shared/utils/appSettingAPIs";
 
 const Activities = ({ location, match }) => {
 
   // cache activity data for updates
-  const { data } = useQuery("activities", fetchActivities);
+  const { data: activitiesData } = useQuery("activities", fetchActivities);
```

Comment:
This change is no longer necessary after I removed the useQuery call for appSettings, but let's just leave it as it's more precise than what was previously there.